www/install.rst

```diff
@@ -79,14 +75,32 @@ popular package managers you can install.
 Macports
 ~~~~~~~~
 
-To install the scipy stack for Python 2.7 with `Macports
+To install the SciPy stack for Python 2.7 with `Macports
 <http://www.macports.org>`_ execute this command in a terminal::
 
     sudo port install py27-numpy py27-scipy py27-matplotlib py27-ipython +notebook py27-pandas py27-sympy py27-nose
 
+Homebrew
+~~~~~~~~
+
+At the time of writing (March 2016), `Homebrew <http://brew.sh/>`_ does not
+have the full SciPy stack available (i.e. you cannot do ``brew install <formula>``
+for everything). However, it is a very popular option (perhaps even more so than
+Macports), and given that active work is being done in this `area <https://github.com/Homebrew/homebrew-python>`_,
+by the `Homebrew group <https://github.com/Homebrew>`_ itself, support for the entire SciPy stack in the future
+seems likely. In the meantime, you can do ``brew install gfortran`` followed ``pip
```

Comment:
I'd remove everything here except the first sentence. This page shouldn't make predictions about the future. The `brew install gfortran` probably won't be enough eg for matplotlib.